src/translations/client.csv

```diff
@@ -156,134 +156,23 @@ changeEmail.validation.msg,Phone number validation message,Must be a valid email
 changePhone.validation.msg,Phone number validation message,Must be a valid 10 digit number that starts with 0,Doit être un numéro valide à {num} chiffres qui commence par {start}.
```

Comment:
I think otherwise this PR is good to go but I'm hesitant of merging this change as this seems a little unrelated. Sure these are not used anymore?

Reply:
Yeah! Reverse checked the messages with the translation script as far as I remember. Also, checked them manually later as well

Reply:
Ok cool! Please solve the conflict and you can merge this